recipes-openxt/openxt/openxt-measuredlaunch/seal-system

```diff
@@ -132,8 +132,10 @@ forward)
     # If PCR8 is all 0 we were not booting with UEFI
     if [ "${tpm2}" -eq 0 ]; then
         pcr8=$(tpm2_listpcrs | grep PCR_08 | cut -f2 -d: | tr -d ' ' | tail -n1)
+        pcr17=$(tpm2_listpcrs | grep PCR_17 | cut -f2 -d: | tr -d ' ' | tail -n1)
```

Comment:
Should this case use `tr '[:upper:]' '[:lower:]'` as well?
If anything maybe for consistency between TPM1.2 vs TPM2.0?

Reply:
`tpm2_listpcrs` already outputs everything in lower-case so it would have no effect

Reply:
My bad... PCR17 seems to be upper case on a TPM1.2 and I did not pay attention this was `tpm2_listpcr` ...

Reply:
More likely it is [`pcr17()`](https://github.com/OpenXT/xenclient-oe/blob/d2bf308089293d3cf88735794188e51d6bcb4345/recipes-openxt/openxt/openxt-measuredlaunch/ml-functions#L170) which has not been modified by this PR, so I'm assuming this is fine.

Reply:
Right, with TPM1.2 it's all uppercase, with `tpm2_listpcrs` it's all lowercase. We only care about that when we check whether it's all "f" (which is why it doesn't matter at all for PCR8 where the check is for 0). `pcr17()` actually doesn't do anything for TPM1.2 devices, I don't really know why it's there. It just takes the current value of the PCR and echoes it back, so we might as well just do `-p17` without specifying an actual value. `¯\_(ツ)_/¯`

Reply:
> I don't really know why it's there

I cannot find out either. It might be that it was doing something else, the [`pcr17()` call site](https://github.com/OpenXT/xenclient-oe/blob/master/recipes-openxt/openxt/openxt-measuredlaunch/ml-functions#L387) passes more arguments than necessary, since its [initial submission](https://github.com/OpenXT/xenclient-oe/commit/2150217c4). Some logic was removed prior introduction I would say.

Nevertheless, out of the scope of this PR :).